pkg/router/f5/f5.go

```diff
@@ -461,6 +481,68 @@ func (f5 *f5LTM) delete(url string, result interface{}) error {
 // Routines for controlling F5.
 //
 
+// ensureVxLANTunnel sets up the VxLAN tunnel profile and tunnel+selfIP
+func (f5 *f5LTM) ensureVxLANTunnel() error {
+	glog.V(4).Infof("Checking and installing VxLAN setup")
+
+	// create the profile
+	url := fmt.Sprintf("https://%s/mgmt/tm/net/tunnels/vxlan", f5.host)
+	profilePayload := f5CreateVxLANProfilePayload{
+		Name:         "vxlan-ose",
+		Partition:    f5.partitionPath,
+		FloodingType: "multipoint",
+		Port:         4789,
+	}
+	err := f5.post(url, profilePayload, nil)
+	if err != nil && err.(F5Error).httpStatusCode != 409 {
+		// error 409 is fine, it just means the tunnel profile already exists
+		glog.V(4).Infof("Error while creating vxlan tunnel - %v", err)
+		return err
+	}
+
+	// create the tunnel
+	url = fmt.Sprintf("https://%s/mgmt/tm/net/tunnels/tunnel", f5.host)
+	tunnelPayload := f5CreateVxLANTunnelPayload{
+		Name:         "vxlan5000",
```

Comment:
Same as above. Can we use a constant for `vxlan5000`? Might be a candidate for a parameter name, maybe?